.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,59 @@
+Please delete the sections without an asterisk if they are not relevant to your PR. All sections with an asterisk are mandatory.
+
+# Description*
+Jira Ticket: SARAALERT-###
+
+Write out a concise summary of this pull request and what it addresses.
+
+# (Feature) Demo
+Insert demo video or photos for a new or updated feature or capability.
+
+# (Bugfix) How to Replicate
+Insert steps for how to replicate the bug this PR addresses.
+
+# (Bugfix) Solution
+Insert description for the solution this PR implements to address the bug.
+
+# Important Changes*
+Please list important files (meaning substantial or integrel to the PR) along with a list of the general changes that should be highlighted for reviewers.
+
+`example_file.js`
+- Example change (ex: refactored import function)
+
+# Testing*
+This fix was tested on the following browsers (submitter must check all those that apply):
+* [ ] Chrome
+* [ ] Firefox
+* [ ] Safari
+* [ ] IE11
+
+Please describe the tests needed to verify your changes. Provide instructions for reproducing these tests. List any relevant details for your test configuration.
+- Test A description
+- Test B description
+
+The following are tests that require visual inspection (remove if not applicable):
+- [ ] 1) Example visual test.
+
+# Submitter Checklist*
+- [ ] I have tested all of my changes locally. 
+- [ ] I have commented my code, particularly in hard-to-understand areas.
+- [ ] I have made corresponding changes to the documentation (if any).
+- [ ] My changes generate no new warnings/errors.
+- [ ] I have added tests that prove my fix is effective or that my feature works.
+- [ ] New and existing unit tests pass locally with my changes.
+
+# Reviewer Checklist*
+## Reviewer 1
+Name: Ron Swanson
```

Comment:
This is funny to type out, but I put these example names here to be ~ fun ~ but we can remove if we're worried about visibility since this is a public repository.

Reply:
It's actually better to @kylietmo someone for each reviewer field. But good examples nonetheless!

Reply:
Yeah good point. Updated to reflect that format.